tests/proxy.test.js

```diff
@@ -148,5 +150,17 @@ test("ssh socks proxy, wrong user", () => {
 });
 
 
+test("ensure logged proxy string does not include any credentials", () => {
+  const testParams = [
+    // [input, expectedOutput]
+    ["socks5://username:password@proxy-host.example.com:9001", "socks5://proxy-host.example.com:9001"],
```

Comment:
```suggestion
    ["socks5://username:password@proxy-host.example.com:9001", "socks5://proxy-host.example.com:9001"],
    ["socks5://username@proxy-host.example.com:9001", "socks5://proxy-host.example.com:9001"],
```

One more test case just to cover our bases